internal/services/containers/container_group_resource_test.go

```diff
@@ -732,6 +732,28 @@ func TestAccContainerGroup_securityContext(t *testing.T) {
 	})
 }
 
+func TestAccContainerGroup_priority(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_container_group", "test")
+	r := ContainerGroupResource{}
+
+	data.ResourceTest(t, r, []acceptance.TestStep{
+		{
+			Config: r.priority(data, "Regular"),
+			Check: acceptance.ComposeTestCheckFunc(
+				check.That(data.ResourceName).ExistsInAzure(r),
+			),
+		},
+		data.ImportStep("ip_address_type"),
+		{
+			Config: r.priority(data, "Spot"),
+			Check: acceptance.ComposeTestCheckFunc(
+				check.That(data.ResourceName).ExistsInAzure(r),
+			),
+		},
+		data.ImportStep("ip_address_type"),
+	})
```

Comment:
could we omit this step given that this property is ForceNew?

Reply:
Testing on the `Sopt` is the main scenario that we want to test against, as by default the priority is `Regular`. I was aware it is ForceNew and these two steps will introduce a re-create, whilst the main thing we want to test is the creation with all possible values of `priority` is successful.